mods/cnc/rules/aircraft.yaml

```diff
@@ -9,14 +9,13 @@ TRAN:
 		BuildPaletteOrder: 10
 		Prerequisites: hpad
 		Queue: Aircraft.GDI, Aircraft.Nod
-	Helicopter:
+	Aircraft:
 		LandWhenIdle: true
 		ROT: 5
 		Speed: 140
 		InitialFacing: 0
 		LandableTerrainTypes: Clear,Rough,Road,Ore,Beach,Tiberium,BlueTiberium
 		AltitudeVelocity: 0c100
-		AirborneUpgrades: airborne
```

Comment:
Not really. The default all of these inherit has that upgrade, so this is just removing junk yaml left by an upgrade rule.